firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -656,6 +656,12 @@ class BrowserViewController: UIViewController,
             executeToolbarActions()
 
             handleMicrosurvey(state: state)
+
+            if state.showPasswordGenerator {
+                if let tab = tabManager.selectedTab {
```

Comment:
Could we combine these two `if` statements into one? Would be easier to read that way.